Author: aurel32
Status: in BTS
Note: config.{guess,sub} has to be updated

diff -u soundtracker-0.6.7/debian/control soundtracker-0.6.7/debian/control
--- soundtracker-0.6.7/debian/control
+++ soundtracker-0.6.7/debian/control
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Junichi Uekawa <dancer@debian.org>
 Standards-Version: 3.6.1
-Build-Depends: debhelper, libgnome-dev, libsndfile1-dev, gettext, libesd0-dev, libgdk-pixbuf-dev, libasound2-dev, dpatch
+Build-Depends: debhelper, libgnome-dev, libsndfile1-dev, gettext, libesd0-dev, libgdk-pixbuf-dev, libasound2-dev [!kfreebsd-i386], dpatch
 
 Package: soundtracker
 Architecture: any

